bootstrap/sql/migrations/native/1.4.0/mysql/postDataMigrationSQLScript.sql

```diff
@@ -0,0 +1,364 @@
+-- Start of Test Definition Parameter Definition Validation Migration
+UPDATE test_definition
```

Comment:
since these are test_definitions would it be easier to it via code then the sql?

Reply:
I was hesitating with this but I though SQL query would be less prone to error. Happy to move them to java.